### PR TITLE
Add landing page for targeting startups

### PR DESCRIPTION
**Pull request by trabaldini**
depends on #117 

The build will be successful again once #117 is merged and this is rebased.

screenshot

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -220,4 +235,4 @@ DEPENDENCIES
   jekyll-seo-tag
 
 BUNDLED WITH
-   1.15.1
+   1.16.0.pre.2
```
Comment (marcoow) [inline Gemfile.lock]: These changes are all unrelated

File: _config.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ name: simplabs.com
 sass:
   sass_dir: css
 
-gems:
+plugins:
```
Comment (marcoow) [inline _config.yml]: This is unrelated

File: disruptors/index.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            <div class="col-md-10 col-md-offset-1 col-xs-12">
+              <h1>&rdquo;</h1>
+              <h4>It has been an absolute pleasure to work with simplabs. Their team of experienced JavaScript engineers quickly slotted in to our in-house development teams. Their expertise with modern, agile software development best practices and tools meant they were able to work with our processes to deliver fantastic customer experiences</h4><br>
+              <h6><strong>Mark Holt, </strong>Trainline CTO</h6>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="row content-section success">
+        <h3>Recent Case Studies</h3>
```
Comment (marcoow) [inline disruptors/index.html]: This header is not needed

File: disruptors/index.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+        <div class="col-sm-6 content-teaser">
+          <h6>ruby on rails, custom engineering, consulting</h6>
```
Comment (marcoow) [inline disruptors/index.html]: We should not be using these tags

File: disruptors/index.html [hunk 5]
```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+title: Disruptors
```
Comment (marcoow) [inline disruptors/index.html]: We should use the same title that we use for the home page here

**Review by marcoow — COMMENTED** [review]
Nice. This should really look the same as the home page though, only with different text and the quote instead of the client logos (without the blue background).

**Review by marcoow — APPROVED** [review]

File: disruptors/index.html [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    </div>
+  </div>
+</div>
+
+<div class="container index landing-page" id="main-section">
+  <div class="row">
+    <div class="col-md-10 col-md-offset-1 col-sm-12">
+      <div class="row content-section separated">
+        <div class="col-sm-12">
+          <p>Disrupting a market is hard enough - <strong>let an experienced partner handle the Engineering for you so you can concentrate on your core business</strong>. Our team of technology experts builds ambitious solutions for clients across the world, from startups to long-established businesses.</p>
```
Comment (pangratz) [inline disruptors/index.html]: Not a native speaker, but the capitalized `Engineering` caught my eye... cc @oliverbarnes @geekygrappler

Comment (oliverbarnes) [inline disruptors/index.html]: **engineering** (lowercase) :)

File: disruptors/index.html [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+      </div>
+      <div class="row content-section success">
+        <div class="col-sm-6 content-teaser">
+          <h4>DD-WRT</h4>
+          <p>DD-WRT is a Linux based <strong>firmware for wireless routers</strong>, running on millions of devices worldwide. We helped embeDD &mdash; the company behind DD-WRT &mdash; <strong>migrate the frontend to a modern tech stack</strong>.</p>
+          <a href="/cases/dd-wrt" class="custom-button">learn more</a>
+        </div>
+        <div class="col-sm-6 content-teaser">
+          <h4>Timify</h4>
+          <p>Timify is an <strong>online appointment scheduling calendar</strong> for businesses that connects service providers with clients. We <strong>helped them to ship a stable, high quality app</strong> that paved the way for them at an early stage.</p>
```
Comment (pangratz) [inline disruptors/index.html]: IMHO `for businesses that connects service providers with clients` reads a bit halting (if that's the right word https://www.dict.cc/?s=holprig). My suggestion is something like:

> Timify is a calendar system which allows clients to schedule an appointment online.

Comment (marcoow) [reply disruptors/index.html]: _" connecting sellers and clients"_ is almost stereotypic startup talk though…

**Review by oliverbarnes — CHANGES_REQUESTED** [review]
Also, the contact footer would also be better as a regular sentence with only the first word capitalized: `Do you have a project in mind?` instead of `Do you Have a Project in Mind?`

File: disruptors/index.html (hunk 6 above)
Comment (marcoow) [reply disruptors/index.html]: fixed

**Comment by marcoow** [comment]
> instead of Do you Have a Project in Mind?

Ouch, wasn't aware we were writing it like that 🤦‍♂️ Will fix in a separate PR.

**Review by oliverbarnes — APPROVED** [review]

File: disruptors/index.html [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
```
Comment (niklaslong) [inline disruptors/index.html]: This is a detail, but I think `We helped them ship a stable, high quality app` reads better than `to ship`.

Comment (marcoow) [reply disruptors/index.html]: agree

**Review by jayjayjpg — APPROVED** [review]
:sparkles: This looks great @EmanuelaTrabaldoLena

File: css/app.scss [hunk 9]
```diff
@@ -130,6 +130,18 @@ blockquote {
...
+  width: 83.33333333%;
+  margin: 0 auto;
+  padding: 0;
+
+  &:before {
+    display: block;
+    content: "\201d";
+  }
+}
+
```
Comment (marcoow) [inline css/app.scss:144]: This is duplicate now: https://github.com/simplabs/simplabs.github.io/blob/master/css/app.scss#L121

File: css/typography.scss [hunk 10]
```diff
@@ -42,6 +42,21 @@ code {
...
+
+  &:before {
+    font-size: 90px;
+    line-height: 30px;
+  }
+
+  cite {
+    font-weight: bold;
+  }
+}
```
Comment (marcoow) [inline css/typography.scss:58]: This is now duplicate as well: https://github.com/simplabs/simplabs.github.io/blob/master/css/typography.scss#L26

**Pull request merged**